cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/AbstractCruiseControlResponse.java

```diff
@@ -53,7 +53,7 @@ public void discardIrrelevantResponse(CruiseControlParameters parameters) {
     }
   }
 
-  String cachedResponse() {
+  public String cachedResponse() {
```

Comment:
Is it something to make the part of the public interface?

Reply:
Originally because it is used in other packages, now moved this method to interface.